Sources/Orbit/Components/CountryFlag.swift

```diff
@@ -49,11 +43,19 @@ public struct CountryFlag: View {
 public extension CountryFlag {
 
     /// Creates Orbit CountryFlag component.
-    init(_ countryCode: String, size: Icon.Size = .normal, border: Border = .default()) {
+    init(_ countryCode: CountryCode, size: Icon.Size = .normal, border: Border = .default()) {
         self.countryCode = countryCode
         self.size = size
         self.border = border
     }
+
+    /// Creates Orbit CountryFlag component with a string country code.
+    /// If a corresponding image is not found, the flag for unkonwn codes is used.
```

Comment:
`unkonwn` typo. The second sentence might be separated by a newline.

Reply:
👍